src/databases/WavefrontOBJ/avtWavefrontOBJWriter.C

```diff
@@ -173,3 +211,56 @@ avtWavefrontOBJWriter::GetCombineMode(const std::string &) const
 {
     return CombineAll;
 }
+
+//****************************************************************************
+// Method:  avtWavefrontOBJWriter::GetColorTable()
+//
+// Purpose:
+//   Create color table
+//
+//
+// Programmer:  Justin Privitera
+// Creation:    11/03/23
+//
+// Modifications:
+//
+//****************************************************************************
+
+vtkImageData *
+avtWavefrontOBJWriter::GetColorTable()
+{
+    const ColorTableAttributes *colorTables = avtColorTables::Instance()->GetColorTables();
+    const ColorControlPointList *table = colorTables->GetColorControlPoints(colorTable);
+    if (table)
+    {
+        unsigned char rgb[256*3];
+        int ncolors = 256;
+        table->GetColors(rgb, ncolors);
+
+        vtkImageData *imageData = vtkImageData::New();
+
+        // TODO change image extent dynamically based on num_ctrl_pts
+        // x: [0, ncolors - 1], y: [0, 0], z: [0, 0]
+        imageData->SetExtent(0, ncolors - 1, 0, 0, 0, 0);
+        imageData->SetSpacing(1., 1., 1.);
+        imageData->SetOrigin(0., 0., 0.);
+        imageData->AllocateScalars(VTK_UNSIGNED_CHAR, 3);
+        unsigned char *pixels = (unsigned char *)imageData->GetScalarPointer(0, 0, 0);
+        unsigned char *ipixel = pixels;
+        for (int i = 0; i < ncolors * 3; i += 3)
+        {
+            *ipixel = rgb[i];
+            ipixel ++;
+            *ipixel = rgb[i + 1];
+            ipixel ++;
+            *ipixel = rgb[i + 2];
+            ipixel ++;
+        }
+
+        return imageData;
+    }
+
+    std::cout << "no table" << std::endl;
```

Comment:
Do you want to leave this output to `stdout` in? I think it might be better as a warning to get shipped back to viewer.

Reply:
Ugh, that's my bad. This shouldn't be here anymore and was just for debugging. Thanks for catching.

Reply:
Removed